closed/make/modules/openjceplus/Lib.gmk

```diff
@@ -43,6 +44,7 @@ else ifeq ($(call isTargetOs, linux), true)
 else ifeq ($(call isTargetOs, windows), true)
   ifeq ($(call isTargetCpu, x86_64), true)
     OPENJCEPLUS_BOOT_JDK := $(call MixedPath,$(OPENJCEPLUS_BOOT_JDK))
+    OPENJCEPLUS_VS_LIB := LIB='$(OPENJ9_VS_LIB)'
```

Comment:
Please reorder these definitions so the names remain sorted.

Reply:
Done.